### ISSUE TITLE
在运用TGS pipeline 生成的vcf文件构建gfa文件时出现问题

### ISSUE THREAD

**Issue by kmosi0** [starskyzheng/panpop#79]
hello！
      I called SV  by TGS pipeline，when convert it to gfa file  for NGS SV recalling，but error  happened.
```
error:[vg::Constructor] Variant/reference sequence mismatch: A vs pos: 24957838: T; do your VCF and FASTA coordinates match?
Variant: 2      24957838        .       A       AAACCCCCCCCCCCCCCCCCCCCCGCCCCCCCCCCACCCCCCCTCCCC        0       .       ZLEN=309;ZORIEND=24958048;ZORIPOS=24957740
zero ind: 24957837 1-indexed: 24957838
```
the vcf ref is not consist with reference genome, the vcf file i used is 15.thin3.vcf.gz,  the same pos ref is not consist in vcf as the picture attached. It seems the error occourred in thin and realign step , the original ref in  01.merge_samples.vcf.gz is right.
![图片](https://github.com/user-attachments/assets/1ff6c361-bbce-459a-aadf-8f58398b5023)
the vcf in  01.merge_samples.vcf.gz
![图片](https://github.com/user-attachments/assets/58187c69-1698-4788-b7ab-00ccbdeef56f)
                                                                                                                                                          Thanks !

**Comment by starskyzheng** [comment]
Could you help trace backwards through the pipeline to locate where the error first occurs?

**Comment by kmosi0** [comment]
It seems the problem occurs in step from 15.thin1.sorted.vcf.gz to 15.thin2.vcf.gz.
![图片](https://github.com/user-attachments/assets/36d960d4-2767-4f38-a1a4-511db9d6a82c)

**Comment by starskyzheng** [comment]
Seems like you used PanPop in TGS mode. This is strange. There should not have overlaped sites for this file. Did you modified the pipeline?

**Comment by kmosi0** [comment]
Sorry for late reply. In order to run this pipeline for both samples with reads and assembly only samples without reads, I modified the pipeline. I set the SVIM-assembly VCF as the merged VCF corresponding to the merged VCF in multi-caller with read samples.
[test.pdf](https://github.com/user-attachments/files/18217137/test.pdf)
, and delete that cal_depth step for assemble only samples.  the dag is attached below. I got that finial vcf file, Is it right? Thanks !

**Comment by starskyzheng** [comment]
DAG was confusing me. There still steps of mapping and SV calling from reads.
You could try the Standalone mode.
`https://github.com/starskyzheng/panpop?tab=readme-ov-file#s_example`

**Comment by kmosi0** [comment]
Yes ,I try the Standalone mode, it's no problem. But more SVs remained without reads depth filter.
